src/libraries/Microsoft.Extensions.Logging/src/LoggerFactoryScopeProvider.cs

```diff
@@ -20,9 +20,9 @@ internal sealed class LoggerFactoryScopeProvider : IExternalScopeProvider
 
         public LoggerFactoryScopeProvider(ActivityTrackingOptions activityTrackingOption) => _activityTrackingOption = activityTrackingOption;
 
-        public void ForEachScope<TState>(Action<object, TState> callback, TState state)
+        public void ForEachScope<TState>(Action<object?, TState> callback, TState state)
```

Comment:
could state be null?

Reply:
`Scope.State` can be, and it's passed in the `callback`